backend/app/api/v1/endpoints/chat.py

```
@@ -1,49 +1,42 @@
             return StreamingResponse(
                 llm_provider.generate_stream_response(messages),
                 media_type='text/event-stream'
             )
 
         ai_response = llm_provider.generate_response(messages)
-        return ResponseHandler.success_response(
-            data={"response": ai_response},
-            message="Chat processed successfully",
-            code=200
-        )
+        return {"response": ai_response}
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)